Adding properties to S6 "Deleted Integer Topology"

danflapjax opened the pull request:
This PR adds seven properties to [S6](https://topology.pi-base.org/spaces/S000006), and two obsoleted/derivable properties were deleted. I also added a second characterization of the space to its description.

prabau commented:
- P17: it's ok.  Or maybe say "The space is a countable sum of copies of ..." 
- P27: use `\mid` (or a colon) instead of `\,|\,`.  Also, there are continuum many open sets.
- P36: the fact that every open set is closed does not prove the space is not connected.  How about being more specific, something like this maybe: "The interval $(0,1)$ is a nonempty proper subset of $X$ that is clopen."
- P121: can you mention the simpler metric from Steen & Seebach instead, and reference that as `See item #6 for space #7 in {{doi:10.1007/978-1-4612-6290-9}}.`
- P87: (not part of your change, but could you also rephrase that one slightly?) Use `{S2}` and `{S194}` at the appropriate place in the text, and I think there is a typo: should be "each of which admits a compatible topological group structure ..."

danflapjax commented:
> P27: use `\mid` (or a colon) instead of `\,|\,`.
I actually just copied the notation from the description of the space, so I've corrected it there too.

> P87: (not part of your change, but could you also rephrase that one slightly?) Use `{S2}` and `{S194}` at the appropriate place in the text, and I think there is a typo: should be "each of which admits a compatible topological group structure ..."

I've implemented the rest of your suggestions, but I'm hung up on which of the following to use for P87. Do you have any thoughts on which would be a better style?

```
The space is the product of {S2} and {S194}, each of which admits a topological group structure. See {S2|P87} and {S194|P87}.
The space is the product of {S2} and {S194}, which admit topological group structures by {T348} and {T349}, respectively.
```

prabau commented:
Looks good to me.
